plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/util/ConstantPoolHelpers.kt

```diff
@@ -68,6 +76,25 @@ internal fun ClassReader.isMinifiedClass(): Boolean {
     return false
 }
 
+/**
+ * See https://github.com/getsentry/sentry-android-gradle-plugin/issues/360
+ * and https://github.com/getsentry/sentry-android-gradle-plugin/issues/359#issuecomment-1193782500
+ */
+/* ktlint-disable max-line-length */
+private val MINIFIED_CLASSNAME_REGEX = """^(((([a-zA-z])\4{1,}|[a-zA-Z]{1,2})([0-9]{1,})?(([a-zA-Z])\7{1,})?)|([a-zA-Z]([0-9])?))(${'\\'}${'$'}((((\w)\14{1,}|[a-zA-Z]{1,2})([0-9]{1,})?(([a-zA-Z])\17{1,})?)|(\w([0-9])?)))*${'$'}""".toRegex()
+
+/**
+ * See https://github.com/getsentry/sentry/blob/c943de2afc785083554e7fdfb10c67d0c0de0f98/static/app/components/events/eventEntries.tsx#L57-L58
+ */
+private val MINIFIED_CLASSNAME_SENTRY_REGEX =
+    """^(([\w\${'$'}]\/[\w\${'$'}]{1,2})|([\w\${'$'}]{2}\/[\w\${'$'}]\/[\w\${'$'}]))(\/|${'$'})""".toRegex()
+/* ktlint-enable max-line-length */
+
+fun classNameLooksMinified(simpleClassName: String, fullClassName: String): Boolean {
```

Comment:
```suggestion
private fun classNameLooksMinified(simpleClassName: String, fullClassName: String): Boolean {
```

Reply:
Can't access it directly from test if I do that.

Reply:
Maybe we add the annotation that is only visible for the test then?

Reply:
I don't think they're available in SAGP (yet).